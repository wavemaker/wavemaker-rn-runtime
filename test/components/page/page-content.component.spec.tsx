import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmPageContent from '@wavemaker/rn-runtime/components/page/page-content/page-content.component';

describe('Test PageContent component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmPageContent name="test_PageContent"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});