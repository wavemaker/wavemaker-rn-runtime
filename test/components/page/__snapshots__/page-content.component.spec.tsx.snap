// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PageContent component Check validity of sample component 1`] = `
<RCTScrollView
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View />
</RCTScrollView>
`;
